tool/gravity/cli/operation.go

```diff
@@ -366,11 +366,40 @@ func getOperationFromBackend(backend storage.Backend) operationGetter {
 	})
 }
 
-func getLocalClusterFromWizard(operator ops.Operator) (cluster *ops.Site, err error) {
-	// TODO(dmitri): I attempted to default to local when creating clusters with wizard
-	// but this breaks when the installer needs to tunnel APIs to the installed cluster
-	// in which case it uses the difference of local (installed cluster) vs non-local
-	// (in wizard state), so resorting to look up
+func getOperationFromWizardBackend(backend storage.Backend) operationGetter {
+	return operationGetterFunc(func() (*ops.SiteOperation, error) {
+		cluster, err := getLocalClusterFromBackend(backend)
+		if err != nil {
+			return nil, trace.Wrap(err)
+		}
+		op, err := storage.GetLastOperationForCluster(backend, cluster.Domain)
+		if err != nil {
+			return nil, trace.Wrap(err)
+		}
+		return (*ops.SiteOperation)(op), nil
+	})
+}
+
+func getLocalClusterFromBackend(backend storage.Backend) (cluster *storage.Site, err error) {
+	// TODO(dmitri): when cluster is created by the wizard, it is not local
+	// so resort to look up
+	clusters, err := backend.GetSites(defaults.SystemAccountID)
```

Comment:
As the comment explains, when the wizard is installing, the cluster record it creates is not local and multiple utility functions that expect the local cluster do not work so it resorts to iterating through the records (which is supposed to be a single anyways).